### PR TITLE
Coding Partner Review

### PR DESCRIPTION
**Pull request by Yordinia**
Hi Prosper Kesssie,

- [ ] Please review the **feature** branch based on the clean code principals of microverse.

### CODE REVIEW

File: modules/draggable.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const a = fromLocal[beingDraggedId];
+  const b = fromLocal[droppedOverId];
+  // using map
+  const list = fromLocal.map((obj, i) => {
+    if (i === beingDraggedId) return b;
+    if (i === droppedOverId) return a;
+    return obj;
+  });
+  updateId(list);
+}
```
Comment (kessie2862) [inline modules/draggable.js]: I will suggest you use the ES6 arrow function syntax for this `finilizeDrag` function

File: modules/draggable.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    isDragOn = false;
+    local = false;
+  } else {
+    isDragOn = true;
+    local = true;
+  }
+  localStorage.setItem('draggable', local);
+
+  return isDragOn;
+}
```
Comment (kessie2862) [inline modules/draggable.js]: As indicated above, I will again suggest you use the ES6 arrow function syntax for this `toggleDrag` function as well.

File: modules/styling-purpose.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    if (hover === false || e.target.value !== '') flag = true;
+    if (flag) return;
+  }
+  input.classList.toggle('on-hover', false);
+}
+
+function hideCursor(e) {
+  e.target.style.cursor = 'none';
+  hover = true;
+}
```
Comment (kessie2862) [inline modules/styling-purpose.js]: Again, kindly use the ES6 arrow function syntax for the `classOn`, `classOff`, and `hideCursor` functions

**Review by kessie2862 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻

## Hi @Yordinia 👋

Good job so far!
There are some issues that you still need to work on but you are almost there!

- [ ] Highly organized code. ✔️
- [ ] GitHub flow is followed. ✔️
- [ ] DRY, YAGNI, and KISS rules are followed.
- [ ] Professional README file. ✔️
- [ ] No linter errors. ✔️

## Changes ♻
Check the comments below and make the required changes.
Submit for a re-review when you're done.

Thanks 😊

File: modules/draggable.js (hunk 1 above)
Comment (Yordinia) [reply modules/draggable.js]: Thanks I will @kessie2862

Comment (Yordinia) [reply modules/draggable.js]: Thanks I will @kessie2862

**Comment by Yordinia** [comment]
Thank @kessie2862 , we'll do.

**Comment by Yordinia** [comment]
Thank @kessie2862 , we'll do.

File: modules/styling-purpose.js (hunk 3 above)
Comment (Yordinia) [reply modules/styling-purpose.js]: Hi @kessie2862 

Unfortunately, I couldn't change all the functions into ES6 arrow Functions.
As you can see below
image
- Changing that into arrow forced me not to use it as a class. (There was an automatic conversion of function to class if it wasn't arror function.)

- Thus,, did not change that one into what you requested. (Even tho  you didn't specificly requested me to change that code block.)

- Im just sharing for conversation and knowledge sake!

 **Thanks**
@Yordinia

**Review by kessie2862 — APPROVED** [review]
Hi @Yordinia 👋

Great job working on the feedback from the previous review. 

# STATUS APPROVED ✔

#### Your project is complete! There is nothing else to say other than... it's time to merge it 🔗 🚢
#### Congratulations! 🎉

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

## Important Notes 📝
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: modules/styling-purpose.js (hunk 3 above)
Comment (kessie2862) [reply modules/styling-purpose.js]: That's fine. It's quite understandable

**Pull request merged**